internal/models/users/access_control/access_control.go

```diff
@@ -0,0 +1,175 @@
+package access_control
+
+import (
+	"database/sql"
+	"errors"
+
+	"github.com/Molnes/Nyhetsjeger/internal/models/users"
+	"github.com/Molnes/Nyhetsjeger/internal/models/users/user_roles"
+	"github.com/lib/pq"
+)
+
+type UserAdmin struct {
+	Email    string
+	IsActive bool // Whether the role is in use or it waits for a user to register (preassigned role)
```

Comment:
I get the gist, but think the `UserAdmin` struct can be described a bit clearer. What does "whether the role is in use" mean? Does it mean that only one person can be `UserAdmin`, because it sounds like it. Is that the same as Organization Admin?

Reply:
changed the comment to `Whether the role is assigned to an existing user or preassigned to an email` in dd8894af6a0f0bad5f5212520838d67508e3ed04